alf/utils/losses.py

```diff
@@ -777,7 +794,7 @@ def forward(self,
             optimal_loss = optimal_loss.mean(-1)
         elif self._reduction == 'sum':
             optimal_loss = optimal_loss.sum(-1)
-        return optimal_loss
+        return optimal_loss, col_ind.squeeze(-1)
```

Comment:
doc-string needs to comment the return values.

Reply:
> doc-string needs to comment the return values.

added